website/src/views/h3n2.ts

```diff
@@ -127,11 +127,7 @@ export class H3n2CompareSideBySideView extends BaseView<
 
         super(
             constants,
-            new CompareSideBySideStateHandler(
-                constants,
-                defaultPageState,
-                organismConfig[constants.organism].pathFragment,
```

Comment:
Some organisms are missing this change - I'll adapt it such that the state handlers simply get the values or their own.